### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/environment-variables.md`

### PR DESCRIPTION
**Pull request by skyanye**

### CODE REVIEW

File: docs/zh/docs/environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
+# 环境变量
+
+/// tip
+
+如果您已经知道什么是“环境变量”以及如何使用它们，请随意跳过此部分。
+
+///
+
+环境变量（有时也被称为" **env var** "）是在种存在于Python代码 **外面** 的变量, 例如在 **操作系统** 中。它可以被Python代码读取，当然也可以被其他程序读取。
```
Comment (nick-cjyx9) [inline docs/zh/docs/environment-variables.md:9]: env var 两侧空格可去，中文单词两侧不用加空格，会降低阅读连续性，中英混排是英语单词两侧加空格。

“例如”翻译不准确

```suggestion
环境变量（有时也被称为"**env var**"）是一种存在于**操作系统**中，Python 代码**之外**的变量。它可以被你的 Python 代码读取（当然也可以被其他程序读取）。
```

File: docs/zh/docs/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+环境变量在处理应用 **设置** 时非常有用，例如在 **安装** Python时。
```
Comment (nick-cjyx9) [inline docs/zh/docs/environment-variables.md:11]: 空格问题

File: docs/zh/docs/environment-variables.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+
+## 创建和使用环境变量
+
+你可以在 **终端或命令行** 中 **创建** 和使用环境变量，而不需要打开Python。
```
Comment (nick-cjyx9) [inline docs/zh/docs/environment-variables.md:15]: duplicated

File: docs/zh/docs/environment-variables.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+
+</div>
+
+////
+
+## 在 Python 中读取环境变量
+
+您还可以在 Python **外部**、在终端中（或使用任何其他方法）创建环境变量，然后 **在Python中读取它们** 。
+
+例如，你可以有一个文件“main.py”，其中包含：
```
Comment (nick-cjyx9) [inline docs/zh/docs/environment-variables.md:57]: 有 -> 新建

File: docs/zh/docs/environment-variables.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+Hello Wade Wilson
+```
+
+</div>
+
+////
+
+## 在 Python 中读取环境变量
+
+您还可以在 Python **外部**、在终端中（或使用任何其他方法）创建环境变量，然后 **在Python中读取它们** 。
```
Comment (nick-cjyx9) [inline docs/zh/docs/environment-variables.md:55]: 空格问题

File: docs/zh/docs/environment-variables.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+Hello Wade Wilson from Python
+```
+
+</div>
+
+////
+
+由于环境变量可以在代码之外设置，同时可以由代码读取。它不必与其余文件一起存储（例如，提交给“git”时）。因此常被用于配置或 **设置** 。
+
+您还可以仅为 **特定程序调用** 创建环境变量，该变量仅对该程序可用。同时，它仅在其设定的时间内可用。
```
Comment (nick-cjyx9) [inline docs/zh/docs/environment-variables.md:132]: 以下仍存在空格问题，不再一一标注

File: docs/zh/docs/environment-variables.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+
+////
+
+当您想学习更多关于[Virtual Environments](virtual-environments.md){.internal-link target=_blank}的知识，请继续阅读。
+
+## 结论
+
+通过上面的内容，您应该对什么是**环境变量**以及如何在 Python 中使用它们有一个基本的了解。
+
+您可以在<a href="https://en.wikipedia.org/wiki/Environment_variable" class="external-link" target="_blank">Wikipedia for Environment Variable</a>了解更多信息。
```
Comment (nick-cjyx9) [inline docs/zh/docs/environment-variables.md:296]: https://zh.wikipedia.org/wiki/%E7%8E%AF%E5%A2%83%E5%8F%98%E9%87%8F

**Review by nick-cjyx9 — CHANGES_REQUESTED** [review]

**Comment by nick-cjyx9** [comment]
@skyanye Can you spend some time to apply these changes?

**Comment by skyanye** [comment]
> @skyanye Could you spend some time applying these changes?

@nick-cjyx9 Sorry, I reject to continue translate anymore. I think the content you marked above does not need to be modified. Because the translation is for people who only know Chinese, in order to express clearly, it is usually not translated directly from English. I saw the translated content on other pages of your website, which is difficult to understand. It is better to read it directly in English. In addition, your website does not parse markdown correctly. If you do not add spaces, there are a lot of asterisks on the website, which greatly affects the reading experience. Please log in to the page and check it yourself.

**Comment by nick-cjyx9** [comment]
Thank you for bringing this to my attention; I will review it later. However, I must respectfully disagree, as I believe the spacing plays an important role in enhancing the reading experience. Inconsistent typesetting can lead to a disorganized presentation, so I won't be able to approve this change.

Apologies for any inconvenience caused.

**Pull request closed without merging**